config/routes.rb

```diff
@@ -1346,6 +1346,24 @@
           save_post
     },
 
+    :physical_rack    =>  {
+      :get  =>  %w(
+        download_data
+        perf_top_chart
+        protect
+        show_list
+        show
+      ) + compare_get,
```

Comment:
Just to make sure, do we support `compare` operations on physical racks?

Reply:
Hmmm, no we don't... I also tried it against a InfraManager provider, PhysicalInfraManager provider, physical servers... none of them supports it even tho they are included in the routes :open_mouth: